notebooks/dataprocessing/module_tests/knn_imputation.ipynb

```diff
@@ -0,0 +1,2246 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# KNNDataImputer Example\n",
+    "\n",
+    "In this example, we will explore how to perform imputation with the HR promotion dataset using KNN imputation. \n",
+    "\n",
+    "This method imputes missing data of a feature using K-nearest neighbours. A feature's missing values are imputed using the mean value from k-nearest neighbors in the dataset. Two samples are close if the features that neither is missing are close. \n",
+    "\n",
+    "This subclass uses the class:`~sklearn.impute.KNNImputer` class from mod:`sklearn` in the background."
```

Comment:
Same problem as with the Iterative imputer notebook: the sentence "class:`~sklearn.impute.KNNImputer` class from mod:`sklearn`" didn't render correctly.